template-processors/applier/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM quay.io/kohlstechnology/eunomia-ocp-templates:latest
+
+USER root
+RUN PACKAGES=".build-deps gcc musl-dev python-dev libffi-dev openssl-dev libpthread-stubs" \
+  && apk add --no-cache --virtual $PACKAGES \
+  && pip install ansible openshift==0.9.0 kubernetes==9.0.0 jmespath \
```

Comment:
Would it be better to switch the versions to variables and put them in ARG or ENV? Future updates would be simpler.

Reply:
@Smiley73 pushed some changes, including some "fancyness" for the image build script and ci.

Reply:
done